static/src/javascripts/projects/common/modules/navigation/navigation.spec.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+import { initNavigation } from 'common/modules/navigation/navigation';
+
+jest.mock('lib/fastdom-promise');
+
+jest.mock('lib/detect', () => ({
+    isIOS: jest.fn(() => false),
+}));
+
+describe('Navigation', () => {
+    const PLACEHOLDER_CLASS = 'js-mega-nav-placeholder';
+    const NAVIGATION_CONTAINER_CLASS = 'navigation-container--default';
+    const LINK_CLASS = 'js-navigation-toggle';
+
+    const domSnippet = `<div class="js-navigation-header navigation-container--collapsed">
+        <a class="${LINK_CLASS}" href="#footer-nav" data-target-nav="js-navigation-header"></a>
+        <div class="${PLACEHOLDER_CLASS}"></div>
+    </div>
+    <div class="js-mega-nav">
+        <div class="global-navigation">Nav</div>
+    </div>
+    <div class="${NAVIGATION_CONTAINER_CLASS}"></div>`;
+
+    beforeEach(() => {
+        if (document.body) {
+            document.body.innerHTML = domSnippet;
+        }
+    });
+
+    afterEach(() => {
```

Comment:
Is this necessary? You reset the DOM beforeEach aswell ...

Reply:
True, I guess from working with `mocha` / `jasmine` / `phantom` in the past I'd always do this to ensure that after the final test had run the DOM was cleaned for subsequent specs. I wondered whether it was necessary with `Jest` so looked elsewhere in our `Jest` specs and this has been done.